Soccar [UI]/src/nl/socnet/message/handler/JoinSessionMessageHandler.java

```diff
@@ -85,4 +98,13 @@ protected JoinSessionMessage decode(Connection connection, ByteBuf buf) throws E
         return new JoinSessionMessage(status, roomName, capacity, settings);
     }
 
+    private void failedJoiningOfSession(String title, String message) {
+        Platform.runLater(() -> {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove useless curly braces around statement (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1602)